keyboards/kingly_keys/romac_plus/rules.mk

```diff
@@ -0,0 +1,49 @@
+MCU = atmega32u4
```

Comment:
Actually, everything between the `MCU` selection and the `BOOTLOADER` selection is no longer needed, and has default values.